spec/models/authorisation_context_spec.rb

```diff
@@ -14,6 +14,14 @@
       end
     end
 
+    context "when user has a matchmaker" do
+      let!(:matchmaker) { create(:matchmaker, user: user) }
+
+      it "returns the FosterParent model" do
```

Comment:
Is the description correct, should it not reference Matchmaker?

Reply:
🤦  yep, that's a C&P error. Fixed now, great spot!